fluss-server/src/main/java/com/alibaba/fluss/server/utils/TableDescriptorValidation.java

```diff
@@ -170,49 +173,68 @@ private static void checkTieredLog(Configuration tableConf) {
         }
     }
 
-    private static void checkAutoPartition(
+    private static void checkPartition(
             Configuration tableConf, List<String> partitionKeys, RowType rowType) {
         boolean isPartitioned = !partitionKeys.isEmpty();
         AutoPartitionStrategy autoPartition = AutoPartitionStrategy.from(tableConf);
-
-        if (!isPartitioned && autoPartition.isAutoPartitionEnabled()) {
-            throw new InvalidConfigException(
-                    String.format(
-                            "Currently, auto partition is only supported for partitioned table, please set table property '%s' to false.",
-                            ConfigOptions.TABLE_AUTO_PARTITION_ENABLED.key()));
```

Comment:
I think this check should still be kept.

Reply:
Done!  I misunderstood.